auth-web/src/components/auth/account-settings/transaction/Transactions.vue

```diff
@@ -173,7 +173,7 @@ export default defineComponent({
     const isLoading = ref(false)
 
     const isTransactionsAllowed = computed((): boolean => {
-      return [Account.PREMIUM, Account.STAFF, Account.SBC_STAFF]
+      return [Account.BASIC, Account.PREMIUM, Account.STAFF, Account.SBC_STAFF]
```

Comment:
This will unhide transactions for basic, there are some other spots to change

Reply:
I think we're good, good change as long is John is ok with it (I think he is)

![image](https://github.com/user-attachments/assets/c39c2d62-df5b-4537-91b4-46012cd4be0c)